app/src/models/editors.ts

```diff
@@ -2,6 +2,7 @@ export enum ExternalEditor {
   Atom = 'Atom',
   VisualStudioCode = 'Visual Studio Code',
   SublimeText = 'Sublime Text',
+  BBEdit = 'BBEdit',
```

Comment:
I think this can just be `BBEdit,`, unless you wrote it like this for clarity.

Reply:
Sadly TypeScript wants them all to be initialized to a value:

<img width="488"  src="https://user-images.githubusercontent.com/359239/31744582-136fa066-b4ab-11e7-946f-c08c8783e90b.png">

Reply:
Oh, right 🤦‍♂️. If you don’t pass a value, it’s assumed to be a number.